host/host.go

```diff
@@ -0,0 +1,38 @@
+package host
+
+import (
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/taskcluster/shell"
+)
+
+func RunCommand(cmd *exec.Cmd) (combinedOutput string, err error) {
+	log.Print("Running command: " + shell.Escape(cmd.Args...))
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Print("Error running command:")
+		log.Print(string(out))
+	}
+	return string(out), err
+}
+
+func RunCommands(allowFail bool, commands ...[]string) error {
```

Comment:
I'm a little confused at the intention here - this plural command seems like it would be a loop around multiple invocations of `RunCommand`, but it takes a slice of strings as its arguments, not `*exec.Cmd`s. Per my comment above, I think we could keep this accepting a slice of strings and convert it to wrap a loop around invocations of `RunCommand`.

Reply:
This method was just moved wholesale from runtime package, I totally agree we should clean up all the utility methods we have around task execution, I just didn't do that in this PR. :-) This is just taking it as it was, and relocating it. But yes, it is pretty ugly, so I can follow up with another PR to address that, it is just a bit more scope creep from the original bug.